include/torch-mlir/Dialect/TorchConversion/Transforms/Passes.h

```diff
@@ -28,7 +28,14 @@ void createTorchBackendToLinalgOnTensorsBackendPipeline(OpPassManager &pm);
 
 /// Creates a pipeline that lowers from the torch backend contract to the
 /// TOSA backend contract.
-void createTorchBackendToTosaBackendPipeline(OpPassManager &pm);
+struct TosaBackendPipelineOptions
+    : public PassPipelineOptions<TosaBackendPipelineOptions> {
+  ListOption<std::string> customOps{
+      *this, "custom-ops",
+      llvm::cl::desc("List of ops to be converted to the backend."),
```

Comment:
nit: I think it's a bit clearer to say "List of ops to be converted to tosa::CustomOp"

Reply:
Done